index.html

```diff
@@ -26,7 +26,7 @@ <h2 class="logo">Mylogo</h2>
     <section class="main">
         <div class="text">
                 <h1>I’m Alexandra.<br>Glad to see you!</h1>
-                <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and
+                <p id="p-1">I’m a software developer! I can help you build a product , feature or website Look through some of my work and
                     experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.
                 </p>
                     <p id="lets-con">LET’S CONNECT</p>
```

Comment:
- [ ] You did an awesome job on your main desktop page! I did notice a small issue though - there appears to be an empty `a` tag located between the **LinkedIn and Medium** icons. To ensure that your page looks as polished and professional as possible, it would be great if you could take a moment to fix this. 🤓

  ### Your design:
  ![image](https://user-images.githubusercontent.com/107952724/236497567-b9893ca7-2731-4252-865f-62fa95cc7b81.png)

- [ ] **[OPTIONAL]** I noticed that your project is currently displaying the default template name instead of your own. **To add a personal touch and help make your project stand out**, you may want to consider replacing the default name with your own. 👌